guides/source/configuring.md

```diff
@@ -1121,6 +1121,11 @@ text/javascript image/svg+xml application/postscript application/x-shockwave-fla
 - `config.active_support.key_generator_hash_digest_class`: `OpenSSL::Digest::SHA1`
 - `ActiveSupport.utc_to_local_returns_utc_offset_times`: `false`
 
+### Configuring Action Text
+
+* `config.action_text.attachment_tag_name` accepts a string for the HTML tag used to wrap attachments. Defaults to 'action-text-attachment'.
+
+
```

Comment:
Let’s move this up above “Results of `config.load_defaults`” with the rest of the framework components.